piedraPapelTijera.html

```diff
@@ -2,49 +2,10 @@
 <html>
```

Comment:
Change file name according to Google file convention: https://developers.google.com/style/filenames